Directory.Build.props

```diff
@@ -43,15 +57,10 @@
   <PropertyGroup>
     <DeveloperBuildTestTfms>netcoreapp3.0</DeveloperBuildTestTfms>
     <StandardTestTfms>$(DeveloperBuildTestTfms)</StandardTestTfms>
-    <StandardTestTfms Condition=" '$(DeveloperBuild)' != 'true' AND '$(OS)' == 'Windows_NT' ">$(StandardTestTfms);net461</StandardTestTfms>
   </PropertyGroup>
 
+  <Import Project="build\repo.props" />
   <Import Project="eng\Dependencies.props" />
   <Import Project="eng\PatchConfig.props" />
   <Import Project="eng\ProjectReferences.props" />
-  <Import Project="eng\Sources.props" />
-
-  <ItemGroup>
-    <PackageReference Include="Internal.AspNetCore.Sdk" PrivateAssets="All" Version="$(InternalAspNetCoreSdkPackageVersion)" />
```

Comment:
I'm assuming we'll need to add a bunch of properties as a result of removing this SDK. For example, this bunch: https://github.com/aspnet/BuildTools/blob/master/src/Internal.AspNetCore.Sdk/build/Common.props#L12-L29.


We need to be careful not to regress important things, like metadata, package icons, etc.